python/dataproc_templates/jdbc/README.md

```diff
@@ -362,6 +364,7 @@ export JARS="<gcs_path_to_jdbc_jar_files>/mysql-connector-java-8.0.29.jar,<gcs_p
 --jdbctogcs.input.url="jdbc:mysql://<hostname>:<port>/<dbname>?user=<username>&password=<password>" \
 --jdbctogcs.input.driver=<jdbc-driver-class-name> \
 --jdbctogcs.input.table=<input table name or subquery with where clause filter> \
+--jdbctogcs.input.sql.query=<input sql query> \ (refer NOTE)
```

Comment:
Lets remove jdbctogcs.input.sql.query from here, as both would never be used together. Below note justifies the usage correctly.